programmers/난이도별/level01.모의고사/JiaLee0707.js

```diff
@@ -0,0 +1,33 @@
+function solution(answers) {
+    const ANSWER_SHEETS = [
+        [1, 2, 3, 4, 5],
+        [2, 1, 2, 3, 2, 4, 2, 5],
+        [3, 3, 1, 1, 2, 2, 4, 4, 5, 5]
+    ];
+    let answerSheetsLength = [];
+    let answerCount = [];
+    ANSWER_SHEETS.forEach(answerSheet => {
+        answerSheetsLength.push(answerSheet.length);
+        answerCount.push(0);
+    });
+   
+    let best = [];
+    
+    answers.forEach((answer, num) => {
```

Comment:
num보단 index가 더 좋겠네요!
```suggestion
    answers.forEach((answer, index) => {
```

Reply:
무심코 넘어가는 실수...
피드백 감사합니다 :)